introduction.md

```diff
@@ -3,3 +3,8 @@ Intro, JavaScript
 This is my intro.
 
 Cheers.
+
+*JS,
+*HTML5,
+*AngularJS.
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?